README.md

```diff
@@ -16,46 +16,69 @@ Great for:
 What is this?
 ---
 
-k3s is a fully compliant Kubernetes distribution with the following changes:
-
-1. Packaged as a single binary.
-1. Lightweight storage backend based on sqlite3 as the default storage mechanism. etcd3, MySQL, Postgres also still available.
-1. Wrapped in simple launcher that handles a lot of the complexity of TLS and options.
-1. Secure by default with reasonable defaults for lightweight environments.
-1. Minimal to no OS dependencies (just a sane kernel and cgroup mounts needed). k3s packages required
-   dependencies
-    * containerd
-    * Flannel
-    * CoreDNS
-    * CNI
-    * Host utilities (iptables, socat, etc)
-    * Ingress controller (traefik)
-    * Embedded service loadbalancer
-    * Embedded network policy controller
+K3s is a [fully conformant](https://github.com/cncf/k8s-conformance/pulls?q=is%3Apr+k3s) Kubernetes distribution with the following changes:
+
+1. It is packaged as a single binary.
+1. It swaps out etcd for a lightweight sqlite3 storage backend. etcd3, MySQL, and Postgres are also supported.
+1. It wraps Kubernetes and other components in a single, simple launcher.
+1. It is secure by default with reasonable defaults for lightweight environments.
+1. It has minimal to no OS dependencies (just a sane kernel and cgroup mounts needed).
+
+K3s bundles the following technologies together into a single cohesive distibution:
+
+* [Containerd](https://containerd.io/) & [runc](https://github.com/opencontainers/runc)
+* [Flannel](https://github.com/coreos/flannel) for CNI
+* [CoreDNS](https://coredns.io/)
+* [Metric-server](https://github.com/kubernetes-sigs/metrics-server)
+* [Traefik](https://containo.us/traefik/) for ingress
+* [Klipper-lb](https://github.com/rancher/klipper-lb) as an embedded service loadbalancer provider
+* [Kube-router](https://www.kube-router.io/) for network policy
+* [Helm-controller](https://github.com/rancher/helm-controller) to allow for CRD-driven deployment of helm manifests
+* [Kine](https://github.com/rancher/kine) as a datastore shim that allows etcd to be replaced with other databases
+* [Local-path-provisioner](https://github.com/rancher/local-path-provisioner) for provisioning volumes using local storage
+* [Host utilities](https://github.com/rancher/k3s-root) such as iptables/nftables, ebtables, ethtool, & socat
+
+Additionally, K3s simplifies Kubernetes operations by maintaining functionality for:
+
+* Managing the TLS certificates of Kubernetes componenents
+* Managing the connection between worker and server nodes
+* Managing an embedded etcd cluster
+* Auto-deploying Kubernetes resources on startup
```

Comment:
I can't think of wording that I love, but it might be worth calling out that it doesn't just deploy on startup - it's constantly deploying the files as they are changed.
```suggestion
* Auto-deploying Kubernetes resources from local manifests, in realtime as they are changed.
```